src/components/app-configure/AppUserLimits.vue

```diff
@@ -64,6 +63,12 @@
               :disabled="storageUnlimited"
             />
           </div>
+          <span
+            class="error-message"
+            :class="storageError ? { show: true } : ''"
+          >
+            Value should not be less than 1 MB and more than 99 GB
```

Comment:
Should we change "and" to "or" here: "The value should not be less than 1 MB or more than 99 GB"?